### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by bratzelk** [bratzelk/Movie-Info#3]
Add some simple unit tests to the program.

**Comment by higginsAmy** [comment]
I would like to work on these.

**Comment by bratzelk** [comment]
That would be great! I think there are a couple (of bad ones) there already to give you an idea.
Just fork the repo and then add a pull request when you're done. Let me know if you need any help.

Thanks,
K.

**Comment by creepystranger** [comment]
Hi Bratzelk I would like to work on this if this is still open

**Comment by higginsAmy** [comment]
Apologies Kim,

I had started working on this last May but got busy with my internship and never pushed my changes up.  Then unfortunately I lost everything when I had to reinstall due to a virus.  I do plan on helping out sometime with the project but I'm afraid I have no changes to push up before someone else works on it.

**Comment by bratzelk** [comment]
@creepystranger Of course, that would be great. Please feel free to add anything you think would benefit the project!

@higginsAmy Please, no apologies required! Your studies should (almost) always come before contributing to open source projects.

**Comment by maiquynhtruong** [comment]
Hi @bratzelk I created a PR for a few test cases that I added. Feel free to review and let me know if there is anything I need to work on further. Thanks
